client/account-recovery/controller.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'lib/analytics';
+import { renderWithReduxStore } from 'lib/react-helpers';
+import DocumentHead from 'components/data/document-head';
+import LostPassword from 'account-recovery/lost-password';
+
+const ANALYTICS_PAGE_TITLE = 'Account Recovery';
+
+export default {
+	lostPassword( context ) {
+		const basePath = context.path;
+
+		analytics.pageView.record( basePath, `${ ANALYTICS_PAGE_TITLE } > Lost Password` );
```

Comment:
Note that we can use:
```
import PageViewTracker from 'lib/analytics/page-view-tracker';
<PageViewTracker path={ basePath } title={ `${ ANALYTICS_PAGE_TITLE } > Lost Password` } />
```

We generally also prefer using the redux analytics events over using the lib directly. (Makes component testing much easier